src/services/ApiCalls.tsx

```diff
@@ -0,0 +1,63 @@
+import { URL_SERVER } from "constants/serverConnexion";
+
+const calculateAge = (birthday: any) => {
+  var ageDifMs = Date.now() - birthday;
+  var ageDate = new Date(ageDifMs); // miliseconds from epoch
+  return Math.abs(ageDate.getUTCFullYear() - 1970);
+};
+interface Patient {
+  firstName: String;
+  lastName: String;
+  age: number;
+  id: String;
+}
+
+export const CallApi = (typeR: string, queryR: any) => {
+  let mkFhir = require("fhir.js");
+  let client = mkFhir({
+    baseUrl: URL_SERVER
+  });
+
+  let patientPromise = client.search({ type: typeR, query: queryR }); // limited to 20 patients ?
+  return patientPromise;
```

Comment:
I'd prefer if you would call this patient and specify the return type of the function (`Promise<Patient>`).
This would make you use this interface you defined ;) (Maybe you could even use it somewhere else)

Reply:
* Patient interface has been centralised 
However, the Promise return an object containing a Fhir Bundle containing patients, it is not directly returning patients